pkgs/misc/cups/libppd.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, cups
+, libcupsfilters
+, ghostscript
+, pkg-config
+, poppler_utils
+, mupdf # mutool
```

Comment:
mutool what?

Reply:
mutool is part of mupdf and picked up by `./configure` - this will be clear in a commit I'm going to push in a moment, where I'll specify the tool paths explicitly which seems to be the better approach (cups-filters did this as well)